_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -41,9 +41,29 @@ PUT /my-index1
   }
 }
 ````
+{% include copy-curl.html %}
 
 In segment replication, the primary shard is usually generating more network traffic than the replicas because it copies segment files to the replicas. Thus, it's beneficial to distribute primary shards equally between the nodes. To ensure balanced primary shard distribution, set the dynamic `cluster.routing.allocation.balance.prefer_primary` setting to `true`. For more information, see [Cluster settings]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-settings/).
 
+Please note that segment replication currently does not support `wait_for requests` in refresh API.
```

Comment:
Is
{: .important }
more appropriate here? Or even 
{: .note }?
The warning tag seems like it should be used for occasions where a user could mess something up. Is it serious business in this case?

Reply:
I think "important" would work well, I'll update it